src/microbe_stage/OrganelleLayout.cs

```diff
@@ -96,6 +99,17 @@ public bool IsTouchingExistingHex(T organelle)
         return false;
     }
 
+    /// <summary>
+    ///     Returns true if the specified organelle is replacing the last hex.
+    /// </summary>
+    public bool IsReplacingLast(T organelle)
+    {
+        if (Count == 1 && (GetOrganelleAt(organelle.Position) != null))
```

Comment:
I think this also should check if the replaced organelle is cytoplasm.

Reply:
Added check in a766ffb